Sources/TuistEnvKit/Settings/VersionProvider.swift

```diff
@@ -16,98 +16,63 @@ protocol VersionProviding {
 }
 
 enum VersionProviderError: FatalError {
-    case dataDecodingError(url: Foundation.URL)
-    case responseError(url: Foundation.URL, content: String, statusCode: Int)
+    case noVersionsError
 
     var description: String {
         switch self {
-        case let .dataDecodingError(url):
-            return "Error decoding the response from \(url.absoluteString) as an utf8 string."
-        case let .responseError(url, content, statusCode):
-            return """
-            The request to \(url.absoluteString) return an unsuccessful response with status code \(statusCode) and error body:
-            \(content)
-            """
+        case .noVersionsError:
+            return "Error fetching versions from GitHub."
```

Comment:
The code has no knowledge that it's actually GitHub so I would use git
```suggestion
            return "Error fetching versions from git."
```